tasks/reduce-snapshot/reduce-snapshot.yaml

```diff
@@ -0,0 +1,50 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: reduce-snapshot
+  labels:
+    app.kubernetes.io/version: "0.0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task to reduce snapshot
+  params:
+    - name: IMAGES
+      type: string
+    - name: SINGLE_COMPONENT
+      type: string
+    - name: CUSTOM_RESOURCE
+      type: string
+    - name: CUSTOM_RESOURCE_NAMESPACE
+      type: string
+      default: ""
+    - name: SNAPSHOT_PATH
+      type: string
+  workspaces:
+    - name: data
+      description: Workspace to save the CR jsons to
+  steps:
+    - name: reduce
+      params:
+        - name: IMAGES
+          value: $(params.IMAGES)
+        - name: SINGLE_COMPONENT
+          value: $(params.SINGLE_COMPONENT)
+        - name: CUSTOM_RESOURCE
+          value: $(params.CUSTOM_RESOURCE)
+        - name: CUSTOM_RESOURCE_NAMESPACE
+          value: $(params.CUSTOM_RESOURCE_NAMESPACE)
+        - name: SNAPSHOT_PATH
+          value: $(params.SNAPSHOT_PATH)
+      ref:
+        resolver: git
+        params:
+          - name: url
+            value: https://github.com/konflux-ci/release-service-catalog
+          - name: revision
+            value: main
```

Comment:
this isn't right

Reply:
production?

Reply:
I think the proper way to do this is to have a task parameter `stepRevision` and `stepRepo` and then in the pipeline we will pass the `taskGitRevision` and `taskGitUrl` values to the task for those parameters. Just like how we do it now for pipelines + tasks, but here the pipeline is the task and the task is the stepAction

Reply:
obsolete comment based on old code.